.travis.yml

```diff
@@ -0,0 +1,30 @@
+language: node_js
+node_js:
+  - '5'
+
+addons:
+  firefox: '43.0'
+
+cache:
+  directories:
+    - '$HOME/.npm'
+    - '$HOME/.cache/bower'
+
+env:
+  - ANGULAR_VERSION=1.2.29
+  - ANGULAR_VERSION=1.3.20
+  - ANGULAR_VERSION=1.4.9
```

Comment:
This package is pointless for 1.5.0+. I set the dependency range to exclude that. See the README

With the method that's used to install the specific version – see `scripts/setBowerResolution.js` – it's not possible to test the bower installation, so there's nothing to test for 1.5.0+